### PR TITLE
Update test_name for test cases

### PR DESCRIPTION
**Pull request by shuheiktgw**
Updated the test case uploader so that it sends test paths instead of test names.

### CODE REVIEW

File: tests/test_client.py [hunk 1]
```diff
@@ -175,20 +181,22 @@ def test_upload_events(self):
             "events": [
                 {
                     "type": "case",
-                    "testName": "test1",
+                    "testPath": ["test1"],
```
Comment (ninjinkun) [inline tests/test_client.py:184]: IIUC, `testPath` is consisted of array of objects like `[{ "type": "class", "name": "ClassA" }, { "type": "case", "name": "test_a" }]`. Does this work correctly with our server?

Comment (shuheiktgw) [reply tests/test_client.py:184]: The code itself works fine, but as you can see [here](https://github.com/launchableinc/nose-launchable/pull/26/files#diff-0d92063e88430a02df61616c5f16b148b64ac4539d9cb9b8d883d5a23351b110R157), I mocked the test path component to return a simple string. That is why the expected body is in a slightly different form. I'll make it similar to the correct one.

**Comment by shuheiktgw** [comment]
@ninjinkun I've fixed it now so would you review the PR again?

**Comment by ninjinkun** [comment]
@shuheiktgw Aha. My concern was unfounded.

**Review by ninjinkun — APPROVED** [review]

**Comment by shuheiktgw** [comment]
Thank you!

**Pull request merged**